torch/testing/_internal/common_methods_invocations.py

```diff
@@ -751,6 +751,25 @@ def to_float(start, end, step):
     yield SampleInput(1, args=(3, 1))
 
 
+def sample_inputs_uniform(op, device, dtype, requires_grad, **kwargs):
+
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=False)
+    samples = (
+        (make_arg((S, S), dtype=dtype, device=device), 0, 1),
+    )
+    for t, hi, lo in samples:
+        yield SampleInput(t, args=(hi, lo))
```

Comment:
NIT: If you put the call to `make_arg` in the yield statement the sample tensor is allocated immediately before yielding it up for use. Rather than allocating all samples ahead of time.  This is a very small number of samples to it is unlikely to be an issue though.

Reply:
done